File: java-advent-of-code/src/test/java/advent/twenty_eighteen/Day13MineCartMadnessTest.java
package advent.twenty_eighteen;

import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class Day13MineCartMadnessTest {
    @Test
    public void checkReal() {
        Day13MineCartMadness im = new Day13MineCartMadness("/twenty_eighteen/Day13-MineCartMadness-input.txt");

        assertEquals("139,65", im.getPart1Answer());
        assertEquals("40,77", im.getPart2Answer());
    }

    @Test
    public void checkExample1() {
        Day13MineCartMadness im = new Day13MineCartMadness("/twenty_eighteen/Day13-MineCartMadness-example#1.txt");

        assertEquals("7,3", im.getPart1Answer());
        assertEquals("9,4", im.getPart2Answer());
    }

    @Test
    public void checkExample2() {
        Day13MineCartMadness im = new Day13MineCartMadness("/twenty_eighteen/Day13-MineCartMadness-example#2.txt");

        assertEquals("2,0", im.getPart1Answer());
        assertEquals("6,4", im.getPart2Answer());
    }
}

File: scala-advent-of-code/src/test/scala/advent/twenty_eighteen/Day13MineCartMadnessTest.scala
package advent.twenty_eighteen

import org.scalatest.{FlatSpec, Matchers}

class Day13MineCartMadnessTest  extends FlatSpec with Matchers {
  "2018 Day 13 - Input" should "calculate answers" in {
    val dp = new Day13MineCartMadness("/twenty_eighteen/Day13-MineCartMadness-input.txt")
    dp.part1Answer should be(0)
    dp.part2Answer should be(0)
  }

  "2015 Day 9 - Example #1" should "calculate answers" in {
    val dp = new Day13MineCartMadness("/twenty_eighteen/Day13-MineCartMadness-example#1.txt")
    dp.part1Answer should be(0)
    dp.part2Answer should be(0)
  }
}
